client/verta/tests/registry/model_version/test_log_dataset.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from verta.dataset import Path
+
+pytestmark = pytest.mark.not_oss  # skip if run in oss setup. Applied to entire module
+
+class TestLogDataset:
+    def test_log_dataset(self, client, model_version, dataset):
```

Comment:
Perhaps we should start moving away from unit tests that make actual API calls to the backend and start mocking them as a general practice?  It seems like our full test suite is very heavy on calls over the network and that makes it painfully slow to run.  I'll let @liuverta weigh in on that, as I'm not sure it needs to be addressed immediately in this PR.

Reply:
This test is alright to me, for now.